packages/boot/tools/supports.ts

```diff
@@ -50,6 +50,11 @@ import { icaMocks, protoMsgMockMap, protoMsgMocks } from './ibc/mocks.js';
 
 const trace = makeTracer('BSTSupport', false);
 
+const cliEntrypoint = await importMetaResolve(
+  'agoric/src/entrypoint.js',
+  import.meta.url,
+).then(u => new URL(u).pathname);
```

Comment:
Generally favor `url.fileURLToPath` for portability.

Reply:
I lean toward [createRequire](https://nodejs.org/api/module.html#modulecreaterequirefilename)

```js
import { createRequire } from 'node:module';
const require = createRequire(import.meta.url);
```